test/Tooltip.spec.js

```diff
@@ -176,7 +176,41 @@ describe('Tooltip', () => {
     wrapper.detach();
   });
 
-  describe('onTimeout', () => {
+  describe('delay', () => {
+    it('should accept a number', () => {
+      isOpen = true;
+      const wrapper = mount(
+        <Tooltip target="target" isOpen={isOpen} toggle={toggle} delay={200}>
+          Tooltip Content
+        </Tooltip>,
+        { attachTo: container }
+      );
+      const instance = wrapper.instance();
+
+      instance.onMouseLeaveTooltip();
+      expect(isOpen).toBe(true);
+      jasmine.clock().tick(200);
+      expect(isOpen).toBe(false);
+    });
+
+    it('should accept an object', () => {
+      isOpen = true;
+      const wrapper = mount(
+        <Tooltip target="target" isOpen={isOpen} toggle={toggle} delay={{ show: 200, hide: 200 }}>
+          Tooltip Content
+        </Tooltip>,
+        { attachTo: container }
+      );
+      const instance = wrapper.instance();
+
+      instance.onMouseLeaveTooltip();
+      expect(isOpen).toBe(true);
+      jasmine.clock().tick(200);
+      expect(isOpen).toBe(false);
+    });
+  });
+
```

Comment:
[this comment](https://github.com/reactstrap/reactstrap/pull/143#issuecomment-247635912) mentions adding a test here for partial delay object. It's fine to add a test case that expects an error, and then fix in another pr or grab the value from the default object which I'd like to eventually be the way this works.